apps/condo/pages/property/[id]/index.tsx

```diff
@@ -207,7 +207,11 @@ const PropertyIdPage: IPropertyIdPage = () => {
     </>
 }
 
-PropertyIdPage.headerAction = <ReturnBackHeaderAction descriptor={{ id: 'menu.AllProperties' }} path={'/property/'}/>
+PropertyIdPage.headerAction = <ReturnBackHeaderAction
+    descriptor={{ id: 'menu.AllProperties' }}
```

Comment:
descriptor here is static, please make it constant.

Reply:
Done